### PR TITLE
Adding LastRunTime and NextRunTime to Manage Page

### PR DESCRIPTION
**Pull request by Syeda-Azim**

### CODE REVIEW

**Review by lweber-d2l — APPROVED** [review]

**Review by lweber-d2l — CHANGES_REQUESTED** [review]
One test needs updating as it is expecting start/end dates:

✖ 1 test failed
FAILED TESTS:
  d2l-manage-schedules
    serialize schedules
      ✖ binds correct values in table
        Chrome Headless 86.0.4240.198 (Linux x86_64)
      AssertionError: expected 'NaN/NaN/NaN NaN:NaN AM' to include '9/1/2020 12:00 AM - 12/31/2020 12:00 AM'
      + expected - actual
    at Context.<anonymous> (test/d2l-manage-schedules.test.js:81:36)

File: src/components/d2l-manage-schedules.js [hunk 1]
```diff
@@ -280,7 +280,8 @@ class ManagerSchedules extends LocalizeMixin(LitElement) {
 				</td>
 				<td>${ this.localize(`schedule.type.${types[schedule.typeId]}`) }</td>
 				<td>${ this.localize(`schedule.frequency.${frequencies[schedule.frequencyId]}`) }</td>
-				<td>${ this._formatDateTime(new Date(schedule.startDate))} - ${this._formatDateTime(new Date(schedule.endDate))}</td>
+				<td>${ this._formatDateTime(schedule.lastRunTime) }</td>
```
Comment (JeremyWinkler) [inline src/components/d2l-manage-schedules.js:283]: Assuming we change lastRunTime to a nullable field like in my comment in the other PR, we would need to update this to account for if it is null.

**Review by lweber-d2l — APPROVED** [review]

File: lang/en.js [hunk 2]
```diff
@@ -40,7 +43,6 @@ export default {
 	'logs.runDate': 'Run Date',
 	'logs.completionDate': 'Completion Date',
 	'logs.completionStatus': 'Completion Status',
-	'logs.unavailableDate': 'N/A',
```
Comment (JeremyWinkler) [inline lang/en.js:43]: The logs page was using this. Could you update the log page to use the new one?
https://github.com/Brightspace/custom-ads-scheduler/blob/master/src/components/d2l-schedule-logs.js#L134

File: src/components/d2l-manage-schedules.js [hunk 3]
```diff
@@ -280,7 +282,8 @@ class ManagerSchedules extends LocalizeMixin(LitElement) {
 				</td>
 				<td>${ this.localize(`schedule.type.${types[schedule.typeId]}`) }</td>
 				<td>${ this.localize(`schedule.frequency.${frequencies[schedule.frequencyId]}`) }</td>
-				<td>${ this._formatDateTime(new Date(schedule.startDate))} - ${this._formatDateTime(new Date(schedule.endDate))}</td>
+				<td>${ this._formatDateTime(schedule.lastRunTime === schedule.nextRunTime ? null : schedule.lastRunTime) }</td>
```
Comment (Syeda-Azim) [inline src/components/d2l-manage-schedules.js:285]: We decided for lastRunTime column to be NOT NULL in DB. So, on Schedule creation we are storing nextRunTime in lastRunTime column and showing N/A in the UI.

Comment (Syeda-Azim) [reply src/components/d2l-manage-schedules.js]: Reverting LastRunTime to nullable column.

**Review by lweber-d2l — APPROVED** [review]

**Pull request merged**